haiku-apps/stopwatch/stopwatch-1.0.0.recipe

```diff
@@ -0,0 +1,42 @@
+SUMMARY="A simple stopwatch application"
+DESCRIPTION="A simple stopwatch application."
+HOMEPAGE="https://github.com/dcatt-haiku/StopWatch"
+COPYRIGHT="2016 DCatt"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/dcatt-haiku/StopWatch/archive/v1.0.0.tar.gz"
+CHECKSUM_SHA256="d17879265ad0a2221e058771ed3d5e78c2207a5b2ec24c8dc226af58171811d4"
+SOURCE_DIR=StopWatch-1.0.0
+
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+
+PROVIDES="
+	stopWatch = $portVersion
+	app:StopWatch = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:g++
+	cmd:make
+	"
+
+BUILD()
+{
+	make $jobArgs OBJ_DIR=objects
```

Comment:
If you use the Makefile template of the makefile-engine, there's no need to convert rdef to rsrc. There's also no need to define a BUILDHOME with it.